document_page/models/document_page_history.py

```diff
@@ -66,11 +66,5 @@ def _get_diff(self, v1, v2):
                 context=True,
             )
 
-    # TODO: Replace for _compute_display_name
     def name_get(self):
```

Comment:
Remove the `name_get()` method, but do NOT remove the `_compute_display_name()` method. That's what is stated in the migration guidelines https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0